docs/requirements.txt

```diff
@@ -3,6 +3,7 @@
 sphinx==4.2.0
 sphinx-rtd-theme==1.0.0
 m2r2==0.3.1
+MarkupSafe==2.0.1 # Version 2.1.0 is not yet compatible with jinja2
```

Comment:
I think setting `jinja2>=3.03` instead is preferable - that version's been out since november last year and 2.11 is quite old :-)

Reply:
Works https://smingdev.readthedocs.io/en/fix-doc-generation/